config.xml

```diff
@@ -3,7 +3,7 @@
     <name>Stockpile</name>
     <description>An app that manages stuff for organizations</description>
     <author email="info@stockpileapp.co" href="https://stockpileapp.co">Stockpile Team</author>
-    <content src="index.html" />
+    <content original-src="index.html" src="http://172.26.74.227:8101" />
```

Comment:
Do you think that might be your computer's external IP address? I did a quick Google search and couldn't find that IP address associated with Ionic CLI, so I think it might be yours.

Can you look into why this was added, and maybe remove it if you can't find anything?